src/test/scala/chiselTests/RangeSpec.scala

```diff
@@ -8,10 +8,64 @@ import org.scalatest.{Matchers, FreeSpec}
 class RangeSpec extends FreeSpec with Matchers {
   "Ranges can be specified for UInt, SInt, and FixedPoint" - {
     "to specify a UInt" in {
-      val x = UInt(range"[0, 7)")
+      val x = UInt(range"[0, 8)")
```

Comment:
Do you want to check edge case [0, 0] in here? Or is the behavior still under debate?